app/config/FrontendAppConfig.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import com.google.inject.{Inject, Singleton}
+import play.api.{Configuration, Environment}
+import play.api.i18n.Lang
+import controllers.routes
+import models.VatEnrolment
+import play.api.mvc.{AnyContent, Request}
+import uk.gov.hmrc.play.config.ServicesConfig
+import utils.PortalUrlBuilder
+
+@Singleton
+class FrontendAppConfig @Inject() (override val runModeConfiguration: Configuration, environment: Environment) extends ServicesConfig with PortalUrlBuilder {
+
+  override protected def mode = environment.mode
+
+  private def loadConfig(key: String) = runModeConfiguration.getString(key).getOrElse(throw new Exception(s"Missing configuration key: $key"))
+
+  private lazy val contactHost = runModeConfiguration.getString("contact-frontend.host").getOrElse("")
```

Comment:
Some of this file is not needed now have a bit of a clean up

Reply:
Cleaned up file from unused methods